client/signup/config/steps-pure.js

```diff
@@ -258,6 +258,8 @@ export function generateSteps( {
 			fulfilledStepCallback: isPlanFulfilled,
 			props: {
 				showBiennialToggle: true,
+				hideFreePlan: true,
+				hideEnterprisePlan: true,
```

Comment:
Why not create a plans intent for this flow and specify the plans there, like done for most other flows. these hideFoo/Bar flags are only used in a couple of older cases and marked as deprecated already. On the list to get rid off, largely obsolete parts